stdlib/date/date_test.go

```diff
@@ -432,6 +432,138 @@ func TestTruncate_Time(t *testing.T) {
 		})
 	}
 }
+
+func Test_Time_with_Time(t *testing.T) {
```

Comment:
nit: following function naming convention, you can rename func to `TestTime_Time, TestTime_Duration`